@xen-orchestra/fs/src/abstract.js

```diff
@@ -209,14 +174,15 @@ export default class RemoteHandlerAbstract {
 
   // write a stream to a file using a temporary file
   async outputStream(
-    input: Readable | Promise<Readable>,
     path: string,
-    { checksum = true, dirMode }: { checksum?: boolean, dirMode?: number } = {}
+    input: Readable | Promise<Readable>,
```

Comment:
I understand this change, but it's used https://github.com/vatesfr/xen-orchestra/blob/819c798e99eb4ca8e5e8b071b966b2435dae4c55/packages/xo-server/src/xo-mixins/backups-ng/index.js#L1377

Reply:
My bad, I see this PR takes care of it :slightly_smiling_face: